examples/github-repo/.rpdk-config

```diff
@@ -1,10 +1,12 @@
 {
     "typeName": "Example::GitHub::Repo",
-    "language": "golang",
+    "language": "go",
     "runtime": "go1.x",
     "entrypoint": "handler",
     "testEntrypoint": "handler",
     "settings": {
-        "importpath": "github.com/aws-cloudformation/cloudformation-cli-go-plugin/examples/github-repo"
+        "importpath": "github.com/aws-cloudformation/cloudformation-cli-go-plugin/examples/github-repo",
```

Comment:
I believe this has changed to "import_path" after the latest changes. Can you please update it?